ga-ccd-definition/ComplexTypes/GAJudicialRequestMoreInfoGAspec.json

```diff
@@ -23,5 +23,13 @@
     "FieldType": "Date",
     "SecurityClassification": "Public",
     "FieldShowCondition": "requestMoreInfoOption=\"REQUEST_MORE_INFORMATION\""
+  },
+  {
+    "ID": "GAJudicialRequestMoreInfoGAspec",
+    "ListElementCode": "endDateForMoreInfoSubmission",
+    "ElementLabel": " ",
+    "FieldType": "Text",
```

Comment:
As mentioned in comment on the other PR, can we please have date as Date and not as a Text to be consistent with the dates that we capture on screen?

Reply:
Done. Please take a look. Cheers!